pkg/accountmanager/manager.go

```diff
@@ -55,8 +55,7 @@ type Interface interface {
 
 type AccountManager struct {
 	client.Client
-	Log logr.Logger
-
+	Log              logr.Logger
```

Comment:
why is Log in caps.. Can be log so that isnt not exposed outside

Reply:
Log is accessed from main.go during initialization.